glossary/achievable.md

```diff
@@ -0,0 +1,4 @@
+An achievable is a single task to be executed successfully, with the least effort. There are two types of achievables:
```

Comment:
achievable is not something that can be executed, but achieved by formulationg a solution that can be executed with a single task